src/api/goods.ts

```diff
@@ -10,10 +10,15 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => {
+      return [...goods].sort((a, b) => a.name.localeCompare(b.name))
+        .slice(0, 6);
+    }); // sort and get the first 5
```

Comment:
```suggestion
        .slice(0, 5);
    }); // sort and get the first 5
```

Reply:
five, not six, array starts from index 0